src/commands/Unban.ts

```diff
@@ -132,13 +132,6 @@ export const DraupnirUnbanCommand = describeCommand({
       },
     }
   ),
-  keywords: {
-    keywordDescriptions: {
-      true: {
-        isFlag: true,
-      },
-    },
-  },
```

Comment:
We should probably keep this, just in case someone does provide it and they get confused why draupnir now spits their command back out (draupnir will give them an error if they provide an unknown option).

Just add a comment above `keywords:` saying

`// This is a legacy option to unban the user from all rooms that we now ignore just so providing the option doesn't cause an error.`